docs/datasheets/dbc-abstracts.md

```diff
@@ -0,0 +1,136 @@
+
```

Comment:
Seems like we are missing the frontmatter.

We are notably also missing license field in the datasheet. It should just describe what we can and can't do with the data if no license is specified.

Reply:
What is the frontmatter?

Reply:
Also, I am not sure how the datasheets should be formatted. I oriented myself according to daradio. But there seem to be vastly different datasheet styles used.

What is the preferred style? And how to put a license that says:

Data may be used to train on and model may be published openly. But data may not be shared with third parties.

Reply:
The frontmatter is a yaml segment at the beginning of a markdown separated by `---`, e.g.

```
---
{yaml}
---
{markdown}
```
In terms of content daradio is quite good, except for the frontmatter + license stuff. For an example of the frontmatter see e.g. [this](https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/docs/datasheets/dagw-gutenberg.md)

the frontmatter is intended for the sheet to be machine readable.

Reply:
Done!